score-lang-cli/src/main/java/org/openscore/lang/cli/SlangCLI.java

```diff
@@ -85,12 +86,12 @@ public String run(
         if (!triggerAsync) {
             StopWatch stopWatch = new StopWatch();
             stopWatch.start();
-            id = scoreServices.triggerSync(compilationArtifact, mergedInputs, systemProperties);
+            id = scoreServices.triggerSync(compilationArtifact, mergedInputs, systemProperties, quiet);
             stopWatch.stop();
-            return triggerSyncMsg(id, stopWatch.toString());
+            return quiet ? StringUtils.EMPTY : triggerSyncMsg(id, compilationArtifact.getExecutionPlan().getName());
         }
```

Comment:
Fixed this. I already added tests for quiet mode. This just printed the flow name instead of the execution duration, but I can't really test this since the duration could be different for different systems.